niceday-api/README.md

```diff
@@ -0,0 +1,37 @@
+# Niceday API Server
+
+## Overview
+This server currently has one POST endpoint `messages` that allows for sending a message
+from the therapist account to a user in the Niceday app.
+
+This server was generated by the [swagger-codegen](https://github.com/swagger-api/swagger-codegen) project.  By using the [OpenAPI-Spec](https://github.com/OAI/OpenAPI-Specification) from a remote server, you can easily generate a server stub.
+
+### Setup
+1. Install npm and node-js, following [these instructions](https://www.npmjs.com/get-npm).
+2. Install dependencies: Run `npm install`.
+3. Run `node login.mjs emailaddress password`, 
+where emailaddress and password should be the credentials for the therapist account. 
+This will print the user id of the therapist as well as the token that you need to authenticate.
+4. Create a file called `.env` in the root of this app.
+Save the therapist user id and token in your `.env` file as THERAPIST_USER_ID and NICEDAY_TOKEN respectively,
+see .env-example. These will be loaded as environment variables and will thus be available in the app.
+
+### Running the server
+To run the server, run:
+
+```
+npm start
+```
+
+To view the Swagger UI interface:
+
+```
+open http://localhost:8080/docs
+```
+
+From there you can easily test out the different endpoints
+
+This project leverages the mega-awesome [swagger-tools](https://github.com/apigee-127/swagger-tools) middleware which does most all the work.
```

Comment:
This project is using `oas3-tools` which is based on `swagger-tools`. It'd be nice to point to `oas3-tools`.

Reply:
Check, kind of stupid, because this sentence was generated by swagger-codegen...